tokio/src/runtime/task/mod.rs

```diff
@@ -201,10 +201,32 @@ use std::{fmt, mem};
 /// [unstable]: crate#unstable-features
 #[cfg_attr(docsrs, doc(cfg(all(feature = "rt", tokio_unstable))))]
 #[cfg_attr(not(tokio_unstable), allow(unreachable_pub))]
-// TODO(eliza): there's almost certainly no reason not to make this `Copy` as well...
-#[derive(Clone, Debug, Hash, Eq, PartialEq)]
+#[derive(Clone, Copy, Debug, Hash, Eq, PartialEq)]
 pub struct Id(u64);
 
+/// Returns the `Id` of the task.
+///
+/// # Panics
+///
+/// This function panics if called from outside a task or if called from a
+/// future passed to `block_on` call.
```

Comment:
I would like more details. Something like "Please note that calls to `block_on` do not have task ids, so the method will panic if called from within a call to `block_on`."